### DISCUSSION TITLE
A tensor with all NaNs was produced Error, does anyone knows how to fix this?

### DISCUSSION THREAD

**Discussion by leonnn1** [vladmandic/sdnext#144] [Q&A]
I was trying to upscale an image 3x, but when I do it I get this error, does anyone knows what it can be?

"NansException: A tensor with all NaNs was produced in VAE. This could be because there's not enough precision to
represent the picture. Try adding --no-half-vae commandline argument to fix this. Use --disable-nan-check commandline
argument to disable this check."

![image](https://user-images.githubusercontent.com/85142472/232182844-92e380f1-3303-4509-979d-6140a75abc91.png)

**Comment by vladmandic** [comment]
its exactly like description says, not sure if i can add much more details than that?

**Reply by leonnn1** [reply]
Well, I don't understand why this happens, specially that it provides a solution to fix it that doesn't work, so what to do is the question, I should have been more clear on my message I think.

**Reply by Krynh** [reply]
> its exactly like description says, not sure if i can add much more details than that?

What does it mean?

**Reply by vladmandic** [reply]
See #374
